tests/nlu/test_train.py

```diff
@@ -116,6 +116,7 @@ def test_all_components_are_in_at_least_one_test_pipeline():
         ), "`all_components` template is missing component."
 
 
+@pytest.mark.timeout(600)
```

Comment:
@wochinge Yes, it would be downloaded the first time(by the first test that needed it) and the model will be cached in a directory for further tests to use it.

Reply:
Awesome 👍